src/TSTransformer/util/wrapExpressionStatement.ts

```diff
@@ -1,16 +1,21 @@
 import luau from "@roblox-ts/luau-ast";
 
-export function wrapExpressionStatement(node: luau.Expression): luau.List<luau.Statement> {
-	if (luau.isTemporaryIdentifier(node) || luau.isNilLiteral(node)) {
-		return luau.list.make();
-	} else if (luau.isCall(node)) {
-		return luau.list.make(luau.create(luau.SyntaxKind.CallStatement, { expression: node }));
+export function wrapExpressionStatement(node: luau.Expression, result: luau.List<luau.Statement>) {
```

Comment:
I don't like that this takes a `luau.List` and mutates it. We should instead create a list internally and return it from this function. Then we can push/unshift that list as needed in the call sites.

Reply:
I remember this not being possible for a specific reason.
It's also not the first function that mutates a parameter list.

Reply:
Hm, tried it and it works fine returning a list. Fixed.